src/jit.c

```diff
@@ -207,15 +207,17 @@ enum operand_size {
 #if defined(_WIN32)
 static const int nonvolatile_reg[] = {RBP, RBX, RDI, RSI, R13, R14, R15};
 static const int parameter_reg[] = {RCX, RDX, R8, R9};
-static const int register_map[] = {
-    RAX, R10, RDX, R8, R9, R14, R15, RDI, RSI, RBX, RBP,
+static struct host_reg register_map[] = {
+    {RAX, 0}, {R10, 0}, {RDX, 0}, {R8, 0},  {R9, 0},  {R14, 0},
+    {R15, 0}, {RDI, 0}, {RSI, 0}, {RBX, 0}, {RBP, 0},
```

Comment:
@jserv, should we do anything to check the casting from `enum x64_reg` (implementation-defined width, usually unsigned integer) to `uint8_t` ?

Reply:
No, by means of bit fields, compilers can help you on type checking.